report/src/main/webapp/assets/sass/_report.scss

```diff
@@ -226,6 +226,91 @@
   }
 }
 
+.toolbar-patterns-list {
+  display: flex;
+  flex-wrap: wrap;
+  padding-left: 5px;
+  padding-right: 5px;
+  padding-bottom: 10px;
+}
+
+.toolbar-patterns-base {
+  border-bottom: 2px solid #dcdce5;
+  padding: 10px;
+  margin: 0 -25px 10px -25px;
+  width: 110%;
+}
+
+.toolbar-patterns-header {
+  font-family: montserratlight, sans-serif;
+  color: #77777c;
```

Comment:
Maybe you could use here some predefined variable? 😃

Reply:
Done, thanks :P